src/js/services/moonpay/moonpay-api.service.js

```diff
@@ -40,6 +40,7 @@ angular
       , uploadFile: uploadFile
       , getFiles: getFiles
       , deleteFile: deleteFile
+      , getUserByIpAddress: getUserByIpAddress
```

Comment:
Should probably be called something like `getCountryByIpAddress` since it gets info about a country, not a user.